presto-hive/src/main/java/com/facebook/presto/hive/s3/HiveS3Config.java

```diff
@@ -374,4 +375,18 @@ public HiveS3Config setS3UserAgentPrefix(String s3UserAgentPrefix)
         this.s3UserAgentPrefix = s3UserAgentPrefix;
         return this;
     }
+
+    @NotNull
+    public PrestoS3AclType getS3AclType()
+    {
+        return s3AclType;
+    }
+
+    @Config("hive.s3.acl.type")
```

Comment:
Since this is only affecting the upload/write path, we can find a better name to reflect it. Maybe `hive.s3.upload-acl-type`.